site/en/guide/basic_training_loops.ipynb

```diff
@@ -521,7 +521,7 @@
         "\n",
         "In this guide, you have seen how to use the core classes of tensors, variables, modules, and gradient tape to build and train a model, and further how those ideas map to Keras.\n",
         "\n",
-        "This is, however, an extremely simple problem.  For a more practical introduction, see [this tutorial](../tutorials/customization/basic_training_loops_walkthrough.ipnyb) that uses real text.\n",
+        "This is, however, an extremely simple problem.  For a more practical introduction, see [this tutorial](../tutorials/customization/custom_training_walkthrough.ipnyb) that uses real text.\n",
```

Comment:
Hmm. These have moved around and not sure where this was intended to point. We can point to the walkthrough but need to remove the "that uses real text"
And please replace "[this tutorial]" with "[Custom training walkthrough]"

Reply:
@lamberta thank you for your review. I have addressed your requested changes.